src/Microsoft.DotNet.Interactive.SqlServer/MsSqlKernel.cs

```diff
@@ -121,6 +121,8 @@ public async Task HandleAsync(SubmitCode command, KernelInvocationContext contex
                 return;
             }
 
+            // If a query handler is already defined, then it means another query is already running in parallel.
```

Comment:
This comment isn't relevant to this change, but there seemed to be some confusion over what this check was for, so I added a comment for it.